src/python/pants/core/goals/test.py

```diff
@@ -85,7 +85,7 @@ class TestResult(EngineAwareReturnType):
     # A None result_metadata indicates a backend that performs its own test discovery/selection
     # and either discovered no tests, or encounted an error, such as a compilation error, in
     # the attempt.
-    result_metadata: ProcessResultMetadata | None
+    result_metadata: ProcessResultMetadata | None  # TODO: Merge elapsed MS of all subproceses
```

Comment:
IMO, it's ok not to do that as long as the text summary of the run makes it clear that the displayed runtime is only for the successful run. Although it could be nice to include both I suppose.

Reply:
I agree it's not essential. I'd like to leave this here, I'd like to think about whether we might still want to gather or aggregate the ProcessResultMetadata.